packages/now-static-build/src/frameworks.ts

```diff
@@ -378,25 +411,48 @@ export const frameworks: Framework[] = [
   {
     name: 'Brunch',
     slug: 'brunch',
+    buildCommand: 'brunch build --production',
     getOutputDirName: async () => 'public',
   },
   {
     name: 'Middleman',
     slug: 'middleman',
     buildCommand: 'bundle exec middleman build',
-    devCommand: 'bundle exec middleman server -p $PORT',
```

Comment:
Why was the dev command removed for middleman?

Reply:
@styfle all devCommands were removed since they are part of @now/frameworks`, so they are not really removed, they'll just be added at a later time

Reply:
@AndyBitz Then why did you add `devCommand: 'nuxt',` above?

Reply:
Good catch, that was a mistake